pkg/sriov/sriov.go

```diff
@@ -127,10 +126,6 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, net
 		return fmt.Errorf("failed to get init netns: %v", err)
 	}
 
-	if len(conf.ContIFNames) < 1 && len(conf.ContIFNames) != len(conf.OrigVfState.HostIFName) {
```

Comment:
i think this  check is related to DPDK.

that said, i dont understand why its here :) perhaps others can shed  some light on it.


what this check means, if container interface name is zero len then we expect the host ifname to be zero len as well (i.e no netdev)

Reply:
That's one way to interpret it. Personally, given the use of len and the word names (here and in other parts), I think that the intention was to compare the amount of names. Assuming that the && was a mistake. It should have been an "or".

In any case, the var is a string and the check is not needed.